newIDE/app/src/UI/Theme/DarkTheme/Mosaic.css

```diff
@@ -23,3 +23,8 @@
 .mosaic-gd-dark-theme.mosaic.mosaic-blueprint-theme .mosaic-preview .mosaic-window-toolbar.draggable:hover {
   background: #3c4698 !important;
 }
+
+/* Separator draggable background */
+.mosaic-gd-dark-theme .mosaic-split.-column {
```

Comment:
Can we do this for all separators and not just for "-column"? Otherwise this does not seems consistent

Reply:
It's ugly, see if I apply the color everywhere.
Especially on the scene editor on the line for moving the blue circles for moving inside scene
![image](https://user-images.githubusercontent.com/1670670/59977226-16838e00-95cf-11e9-8d59-ca89f0c4a749.png)

Reply:
Indeed, it's not pretty. I wonder if the solution is just to add a title bar (without a close button) to the area where events are shown?

Reply:
Or the put the funciton configuration at the bottom, so that the title bar "Function Configuration" is used for delimiting the two areas. But would be a bit strange to have it below events, seems counter intuitive.

Reply:
- With a title bar we lose space.
- With the window at the bottom it's too strange and yes its counter intuitive.
-I think that between moving and adding code, adding a CSS rule is much lighter and customizable.